### PR TITLE
Feature/fetch images

### PR DESCRIPTION
**Pull request by collinkallery**
### Type of change made:

- [x] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Styling (Styling to new or existing files)

### Detailed Description
- Implements the use of the opengraph API to fetch the image from each article's original URL to use as a preview on our website. 
- Creates the article container. 
- In the article preview, fetches the image from the URL and places it into the rendering utilizing a React hook. 
### Why is this change required? What problem does it solve?
- This change is required because it allows a user to have more of a visually pleasing experience compared to the original Hacker News because they are able to see images rather than just bland text across the website. 
### Was any previously written code changed?
- Removal of CSS files 
### Directory modified:

- [x] - App
- [ ] - Article
- [x] - ArticleContainer
- [ ] - ArticleExpanded
- [x] - ArticlePreview
- [ ] - Comment
- [ ] - CommentContainer
- [ ] - Login
- [ ] - NavBar
- [ ] - UserInfo
- [ ] - WelcomePage

### CODE REVIEW

File: src/Components/ArticleContainer/ArticleContainer.js [hunk 1]
```diff
@@ -11,13 +11,20 @@ const ArticleContainerStyled = styled.div`
...
 
-  return <ArticleContainerStyled>{articlesToDisplay}</ArticleContainerStyled>;
-};
+const ArticleContainer = (props) => {
+  const articlesToDisplay = props.homePageStories.map(story => {
+    console.log('story', story);
+    return (
+      <ArticlePreview {...story} key={story.id}/>
+    )
+  })
```
Comment (collinkallery) [inline src/Components/ArticleContainer/ArticleContainer.js:22]: maps over homepage articles, creates article previews for each

File: src/Components/ArticlePreview/ArticlePreview.js [hunk 2]
```diff
@@ -1,19 +1,38 @@
...
+  const [description, setDescription] = useState(null);
+
+  fetchImage(props.url)
+    .then(data => {
+      return (
+        setImage(data.hybridGraph.image),
+        setDescription(data.hybridGraph.description)
+      )
+    })
+
```
Comment (collinkallery) [inline src/Components/ArticlePreview/ArticlePreview.js:28]: We'll change this method name maybe to "fetch details"  - this is fetching the details of each URL using the opengraph API.

File: src/Components/ArticlePreview/ArticlePreview.js [hunk 3]
```diff
@@ -1,19 +1,38 @@
...
   return (
-    <StoryCard>
-      <p>{props.title}</p>
-    </StoryCard>
-  );
-};
+      <StoryCard>
+        <p>{props.title}</p>
+        <img src={image} />
+        <p id={props.id}>{description}</p>
```
Comment (collinkallery) [inline src/Components/ArticlePreview/ArticlePreview.js:33]: ID to eventually be used for a label?

File: src/Components/ArticlePreview/ArticlePreview.js (hunk 2 above)
Comment (laurenlucero) [reply src/Components/ArticlePreview/ArticlePreview.js:28]: that's a good name, more semantic

File: src/Components/ArticlePreview/ArticlePreview.js (hunk 3 above)
Comment (laurenlucero) [reply src/Components/ArticlePreview/ArticlePreview.js:33]: yes, also need to put an alt on the img

**Comment by C-Bandstra** [comment]
Thank you so much for all the comments!!

**Pull request merged**